CONTRIBUTING.md

```diff
@@ -151,13 +152,13 @@ rules on your computer. We will now walk you through the steps to get going:
 
     ```sh
     npm install configurable-http-proxy
-    export PATH=$(pwd)/node_modules/.bin:$PATH
+    export PATH=/<path>/<to>/node_modules/.bin:$PATH
```

Comment:
If following the command `npm install configurable-http-proxy`, the path to node_modules will be from the local directory as known by `$(pwd)`.

Due to that I think what we have already is quite good.